src/app/reports/report-card.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { DomSanitizer } from "@angular/platform-browser";
+
+import { MessagingService } from "jslib-common/abstractions/messaging.service";
+import { StateService } from "jslib-common/abstractions/state.service";
+
+export enum ReportTypes {
+  "exposedPasswords" = "exposedPasswords",
+  "reusedPasswords" = "reusedPasswords",
+  "weakPasswords" = "weakPasswords",
+  "unsecuredWebsites" = "unsecuredWebsites",
+  "inactive2fa" = "inactive2fa",
+  "dataBreach" = "dataBreach",
+}
+
+type ReportEntry = {
+  title: string;
+  description: string;
+  route: string;
+  icon: string;
+  requiresPremium: boolean;
+};
+
+const reports: Record<ReportTypes, ReportEntry> = {
+  exposedPasswords: {
+    title: "exposedPasswordsReport",
+    description: "exposedPasswordsReportDesc",
+    route: "exposed-passwords-report",
+    icon: `
```

Comment:
Is there any specific reason why we're storing the `svg` data inline instead of saving the image in the project and pathing to it when needed? Would doing something like that change your card architecture too much?

Reply:
Primarily to be able to use currentColor. Only alternative is to use embeed which messes up centering of the svgs.